ui/app/models/dynamic-host-volume.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Model from '@ember-data/model';
+import { attr, belongsTo, hasMany } from '@ember-data/model';
+export default class DynamicHostVolumeModel extends Model {
+  @attr('string') plainId;
+  @attr('string') name;
+  @attr('string') path;
+  @attr('string') namespace;
+  @attr('string') state;
+  @belongsTo('node') node;
+  @attr('string') pluginID;
+  @attr() constraints;
+  @attr('date') createTime;
+  @attr('date') modifyTime;
+  @hasMany('allocation', { async: false }) allocations;
+  @attr('string') accessMode;
+  @attr('string') attachmentMode;
```

Comment:
CSI volumes have a list of "requested capabilities" and a single "capabilities" which reflects the current claim on the volume. 

But dynamic host volumes have only "requested capabilities" which are a list of access mode / attachment mode pairs: https://github.com/hashicorp/nomad/blob/f3d53e3e2b4c96f5c3e87f5bcb9a1a48ec9e956b/website/content/api-docs/volumes.mdx#sample-response-4 and all of them are of interest to job authors on the volume detail page.

This ends up being reflected in the shape of the table here: https://nomad-dnblzzfqu-hashicorp.vercel.app/ui/storage/volumes/dynamic/array-0@default and in the hashing in the serializer in `./ui/app/serializers/dynamic-host-volume.js`